wf-core-tck-runner/run_all.sh

```diff
@@ -20,6 +20,67 @@ if [[ -z "${JBOSS_HOME}" ]]; then
   exit 1
 fi
 
+# locate SPEC API jars if not already specified
+if [[ -z "${ANNOTATION_API}" ]]; then
+  pushd ${JBOSS_HOME}
+  export ANNOTATION_API=`find * -name jakarta*annotation*api*.jar`
```

Comment:
Does this work with the embedded `\n` new lines?

Reply:
which `\n` new lines do you mean?

Reply:
When you do a `find` it will print each found file on a new line. You'd end up with the environment variable looking something like:
```
ANNOTATION_API=some/path/cool.jar
some/path/other.jar
some/path/another.jar
```
I would think only the first one, `cool.jar` in the example, would be seen.

Reply:
If multiple jars match the pattern than we need to update the find expression.   That sounds like a condition we should check for

Reply:
Ah, I guess all these do just return a single JAR. Maybe something we more need to keep an eye on then.